demos/expand-component/index.html

```diff
@@ -0,0 +1,46 @@
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" />
+  <title>2025 Esri Developer & Technology Summit - Expand Component Focus Trap Disabled</title>
+
+  <!-- Calcite imports -->
+  <script type="module" src="https://js.arcgis.com/calcite-components/3.0.3/calcite.esm.js"></script>
+
+
+  <!-- ArcGIS Maps SDK for JavaScript imports -->
+  <script src="https://js.arcgis.com/4.32/"></script>
+  <link id="jsapi-mode-light" rel="stylesheet" href="https://js.arcgis.com/4.32/esri/themes/light/main.css" />
+  <link disabled id="jsapi-mode-dark" rel="stylesheet" href="https://js.arcgis.com/4.32/esri/themes/dark/main.css" />
+  <script type="module" src="https://js.arcgis.com/map-components/4.32/arcgis-map-components.esm.js"></script>
+
+  <!-- Demo imports -->
+  <script src="./app.js" defer></script>
+  <link rel="stylesheet" href="./style.css" />
+</head>
+
+<body>
+  <calcite-shell content-behind>
+    <calcite-navigation slot="header" id="nav">
+      <calcite-navigation-logo href="#" icon="accessibility" alt="Application logo" slot="logo"
+        heading="Expand Component Focus Trap Disabled"
+        description="Esri Developer & Technology Summit 2025"></calcite-navigation-logo>
+      <calcite-action-pad layout="horizontal" expand-disabled slot="content-end">
+        <calcite-action id="toggle-mode" text="Toggle mode" icon="brightness"></calcite-action>
+      </calcite-action-pad>
+
+      <calcite-tooltip placement="bottom" reference-element="toggle-mode" slot="content-end">Toggle
+        mode</calcite-tooltip>
+    </calcite-navigation>
+    <arcgis-map item-id="05e015c5f0314db9a487a9b46cb37eca">
+      <arcgis-home position="top-right"></arcgis-home>
+      <arcgis-expand id="expand-el" focus-trap-enabled="false" close-on-esc position="top-right">
```

Comment:
I think we should add mode="floating" here. Otherwise in small screen sizes when the search expands it docks to the bottom and there is a lot of white space.